coin-flip/index.css

```diff
@@ -3,25 +3,30 @@ body {
     background: #282b29;
     color: #70b9e6;
     padding: 15px;
-  }
-  .header {
+    text-align: center;
```

Comment:
Instead of using a `<center>` tag, I'm using `text-align: center` to center your content. Make sure things are properly centered how you wish.

Reply:
Nice! I knew center tags were bad practice I just didn't know how else you could center something. Thanks